uelc/main/views.py

```diff
@@ -208,7 +208,10 @@ def notify_facilitators(self, request, path, notification):
                  content=json.dumps(msg))
 
         socket.send(json.dumps(e))
-        socket.recv()
+        try:
+            socket.recv(zmq.NOBLOCK)
+        except zmq.ZMQError:
+            pass
```

Comment:
@djredhand this change here makes `socket.recv()` non-blocking. Without this, the selenium test for the decision block hangs. If this needs to be blocking for the right functionality, let me know and I'll figure out another way to do this.